src/settings/JobsSettings/constants.js

```diff
@@ -0,0 +1,21 @@
+import { EXPORT_JOB_TYPE_KEYS } from '../../ExportJobs/constants';
+
+const {
+  AUTH_HEADINGS_UPDATES,
+  BURSA_FEES_FINES,
```

Comment:
The `BURSA_FEES_FINES` import is correct, but it's a typo. See [ExportJobs/constants.js#L5](https://github.com/folio-org/ui-export-manager/blob/a991408aef984cfb8daed9a61212d0e3da825b25/src/ExportJobs/constants.js#L5). Correct it here, or in a separate PR. Yeah, I know, pedantic, but it's still a typo.